storybook/src/components/banner/banner.css-template.ts

```diff
@@ -18,11 +18,7 @@ export const cssBanner: ComponentImplementation<Banner<TemplateResult>> = {
           })}
           role="alert"
         >
-          <div class="container">
-            <div class="row">
-              <div class="col-sm-12">${content}</div>
-            </div>
-          </div>
+          <div class="dso-banner-inner">${content}</div>
```

Comment:
Als ik door onze templates zoek kom ik nu alleen in deze `css-template` en de `core-template` een className tegen met `-inner` erin.

Het lijkt erop dat de conventie is  om `-container` te gebruiken.

Zou je `dso-banner-inner` willen vervangen door `dso-banner-container`?

Reply:
Zie boven

Reply:
Prima. Laten we het zo. 👍